src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
+    transform-origin: bottom center;
+    animation: rotate 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $hand-width-seconds;
+    background-color: $seconds-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-seconds} / 2));
```

Comment:
Similar to the previous comment, consider using a simpler method like `transform: translateX(-50%)` combined with `left: 50%` to center the seconds hand without the need for the `calc` function, for better performance and readability.